### PR TITLE
Rename gw uplink

### PR DESCRIPTION
**Pull request by aligeti**
renaming the gateway tool to be the uplink tool

---
This change is [Reviewable](https://reviewable.io/reviews/storj/storj/273)

### CODE REVIEW

File: cmd/uplink/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Gateway
```
Comment (jtolds) [inline cmd/uplink/README.md]: uplink?

File: go.sum [hunk 2]
```diff
@@ -59,7 +59,7 @@ github.com/fatih/color v1.7.0 h1:DkWD4oS2D8LGGgTQ6IvwJJXSL5Vp2ffcQg58nFV38Ys=
 github.com/fatih/color v1.7.0/go.mod h1:Zm6kSWBoL9eyXnKyktHP6abPY2pDugNf5KwzbycvMj4=
 github.com/fatih/structs v1.0.0 h1:BrX964Rv5uQ3wwS+KRUAJCBBw5PQmgJfJ6v4yly5QwU=
 github.com/fatih/structs v1.0.0/go.mod h1:9NiDSp5zOcgEDl+j00MP/WkGVPOlPRLejGD8Ga6PJ7M=
-github.com/fsnotify/fsnotify v1.4.7 h1:IXs+QLmnXW2CcXuY+8Mzv/fWEsPGWxqefPtCP5CnV9I=
+github.com/fsnotify/fsnotify v1.4.7 h1:IXs+QLmnXW2CcXuY+8Mzv/fWEsPULxqefPtCP5CnV9I=
```
Comment (jtolds) [inline go.sum]: what happened here? should this file change? i don't think this file should change

**Comment by jtolds** [comment]
good job

File: cmd/uplink/README.md (hunk 1 above)
Comment (aligeti) [reply cmd/uplink/README.md]: Done

File: go.sum (hunk 2 above)
Comment (aligeti) [reply go.sum]: Done

**Pull request merged**